src/main/resources/static_config/static_action_groups.yml

```diff
@@ -130,6 +131,8 @@ cluster_composite_ops_ro:
   - "indices:admin/aliases/get*"
   - "indices:data/read/scroll"
   - "indices:admin/resolve/index"
+  - "indices:data/read/point_in_time/read*"
```

Comment:
What does `read*` mean, Is it read all pits if yes should we name it different to avoid confusion with index data read